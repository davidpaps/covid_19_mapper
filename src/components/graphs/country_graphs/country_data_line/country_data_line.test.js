import * as React from "react";
import { shallow } from "enzyme";
import CountryLineData from "./country_data_line";

describe("Country Line", () => {
  let wrapper;

  beforeEach(
    () =>
      (wrapper = shallow(
        <CountryLineData
          createLineLabels={[]}
          country={""}
          data={[]}
          total={[]}
          countries={[]}
        />
      ))
  );

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(2);
  });

  it("should render a <Line />", () => {
    expect(wrapper.find("Line").length).toEqual(1);
  });

  it("should render a <Button />", () => {
    expect(wrapper.find("Button").length).toEqual(1);
  });
});
