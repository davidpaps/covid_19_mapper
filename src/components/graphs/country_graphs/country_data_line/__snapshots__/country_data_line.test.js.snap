// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country Line renders correctly 1`] = `ShallowWrapper {}`;
